src/postgres/src/test/regress/expected/yb_pg_foreign_key.out

```diff
@@ -1757,62 +1860,9 @@ create schema fkpart0
       (foreign key (a) references fkpart0.pkey) for values in (2, 3)
       partition by list (a)
   create table fk_part_23_2 partition of fk_part_23 for values in (2);
-
-alter table fkpart0.fk_part add foreign key (a) references fkpart0.pkey;
-\d fkpart0.fk_part_1	\\ -- should have only one FK
-alter table fkpart0.fk_part_1 drop constraint fk_part_1_a_fkey;
-
-\d fkpart0.fk_part_23	\\ -- should have only one FK
```

Comment:
Did you remove these by mistake?
Lets keep these tests as commented.

Reply:
Done!